docs/tutorials/extending_admin_ui/2_creating_a_dashboard_tab.md

```diff
@@ -0,0 +1,137 @@
+# Creating a Dashboard tab
+
+The Dashboard is the front page that you visit after logging in to the Back Office.
+By default it contains two blocks: "Me" and "Everyone", which list Content items and Media.
+The "Me" block additionally lists your own drafts.
+
+![Unmodified dashboard](img/dashboard.png)
+
+In this step you will add a new tab to the "Everyone" block in the Dashboard.
+This tab, called "Articles", will list ten most recently modified Content items of the Content Type `article`.
+
+!!! tip
+
+    To be able to view the results of this step, create a few Content items of the type "Article".
+
+## Register a service
+
+First, add the following block to `src/EzSystems/ExtendingTutorialBundle/Resources/config/services.yml`,
+which was created when [the bundle was generated](1_creating_a_bundle.md):
+
+``` yml
+EzSystems\ExtendingTutorialBundle\Tab\Dashboard\Everyone\EveryoneArticleTab:
+    autowire: true
+    autoconfigure: true
+    public: false
+    tags:
+        - { name: ezplatform.tab, group: dashboard-everyone }
```

Comment:
I would add `services:` at the beginning to keep the structure of this file. I've pasted the code and didn't care to make spaces and I've received some errors.

Reply:
Added note about it above the code block (also in the two other cases below)